tests/vm_test.go

```diff
@@ -423,8 +423,8 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			}, 300*time.Second, 1*time.Second).Should(BeTrue())
 		},
 			table.Entry("with ContainerDisk", newVirtualMachineInstanceWithContainerDisk),
-			table.Entry("[Serial][rook-ceph]with OCS Filesystem Disk", newVirtualMachineInstanceWithOCSFileDisk),
-			table.Entry("[Serial][rook-ceph]with OCS Block Disk", newVirtualMachineInstanceWithOCSBlockDisk),
+			table.Entry("[Serial][block-storage]with Filesystem Disk", newVirtualMachineInstanceWithFileDisk),
```

Comment:
The `[block-storage]` conflicts with the `newVirtualMachineInstanceWithFileDisk` on the same line. I think you can remove the block storage filtering

Reply:
right although them this test wont run with OCS filesystem since it wont run in the storage lane.. same for all those tests you commented this comment.. WDYT?

Reply:
Right but it would then run in non 'storage' lanes. maybe we need a `[storage]` for filesystem storage tests?